workflows/taxonomic_classification/README.md

```diff
@@ -12,43 +12,79 @@
 mkdir data
 cd ~/data
 curl -O https://s3-us-west-1.amazonaws.com/spacegraphcats.ucdavis.edu/microbe-refseq-sbt-k51-2017.05.09.tar.gz
+curl -O https://s3-us-west-1.amazonaws.com/spacegraphcats.ucdavis.edu/microbe-refseq-sbt-k31-2017.05.09.tar.gz
+curl -O https://s3-us-west-1.amazonaws.com/spacegraphcats.ucdavis.edu/microbe-refseq-sbt-k21-2017.05.09.tar.gz
 curl -O https://s3-us-west-1.amazonaws.com/spacegraphcats.ucdavis.edu/microbe-genbank-sbt-k51-2017.05.09.tar.gz
+curl -O https://s3-us-west-1.amazonaws.com/spacegraphcats.ucdavis.edu/microbe-genbank-sbt-k31-2017.05.09.tar.gz
+curl -O https://s3-us-west-1.amazonaws.com/spacegraphcats.ucdavis.edu/microbe-genbank-sbt-k21-2017.05.09.tar.gz
+
 tar xzf microbe-refseq-sbt-k51-2017.05.09.tar.gz
+tar xzf microbe-refseq-sbt-k31-2017.05.09.tar.gz
+tar xzf microbe-refseq-sbt-k21-2017.05.09.tar.gz
 tar xzf microbe-genbank-sbt-k51-2017.05.09.tar.gz
+tar xzf microbe-genbank-sbt-k31-2017.05.09.tar.gz
+tar xzf microbe-genbank-sbt-k21-2017.05.09.tar.gz
+
 rm -r microbe-refseq-sbt-k51-2017.05.09.tar.gz
+rm -r microbe-refseq-sbt-k31-2017.05.09.tar.gz
+rm -r microbe-refseq-sbt-k21-2017.05.09.tar.gz
 rm -r microbe-genbank-sbt-k51-2017.05.09.tar.gz
-```
-#### Now, download the containers
-```
-docker pull quay.io/biocontainers/sourmash:2.0.0a1--py35_2
-docker pull quay.io/biocontainers/kaiju:1.5.0--pl5.22.0_0
-docker pull quay.io/biocontainers/kraken:0.10.6_eaf8fb68--pl5.22.0_4
+rm -r microbe-genbank-sbt-k31-2017.05.09.tar.gz
+rm -r microbe-genbank-sbt-k21-2017.05.09.tar.gz
 ```
 #### If you don't already have the trimmed data you can download it from osf using the following command. First, you will need to copy trimmed_files.txt to your working directory or provide the path to this file. 
 ```
-for i in $(cat trimmed_2_files.txt) 
-do 
-	osf -u philliptbrooks@gmail.com -p dm938 fetch osfstorage/data/${i} 
-done 
+for i in $(cat trimmed_{"2","30"}_files.txt)
+do
+	osf -u <username> -p dm938 fetch osfstorage/data/${i}
```

Comment:
My inner paranoid security nut would be much happier if hard-coded credentials were no longer needed... Maybe we could remove it from the formal documentation but have a "memo" file with an explanation of why hard-coded credentials might be needed and how to use them?

Reply:
Agreed! For now, I think we should avoid using the osf cli. Alternatively, we could provide the urls and use curl or wget.